arcgis-ios-sdk-samples/Features/List related features/ListRelatedFeaturesViewController.swift

```diff
@@ -32,7 +32,7 @@ class ListRelatedFeaturesViewController: UIViewController, AGSGeoViewTouchDelega
         (self.navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = ["ListRelatedFeaturesViewController", "RelatedFeaturesListViewController"]
         
         // initialize map with a basemap
```

Comment:
I noticed you updated most of the other comments. Not sure if you forgot about this one!